curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/dictionaries-and-loops.md

```diff
@@ -26,25 +26,22 @@ for key in counts:
 
 ## --answers--
 
-annie 42
-
-jan 100
+<pre>annie 42
+jan 100</pre>
 
 ---
 
-chuck 1
-
+<pre>chuck 1
 annie 42
-
-jan 100
+jan 100</pre>
 
 ---
 
-chuck 1
+<pre>chuck 1</pre>
 
 ---
 
-\[Error]
+<pre>\[Error]</pre>
```

Comment:
All of the added `\` to escape need to be removed, if they are within `pre` tags:
![image](https://user-images.githubusercontent.com/51722130/104013659-aa18b180-51a9-11eb-837c-3e18bb87ef2e.png)

Live site: https://www.freecodecamp.org/learn/scientific-computing-with-python/python-for-everybody/dictionaries-and-loops